platform-integration/src/main/java/com/softicar/platform/integration/database/structure/version/DatabaseStructureVersionResources.java

```diff
@@ -0,0 +1,57 @@
+package com.softicar.platform.integration.database.structure.version;
+
+import com.softicar.platform.common.core.exceptions.SofticarDeveloperException;
+import com.softicar.platform.common.io.resource.supplier.IResourceSupplier;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class DatabaseStructureVersionResources {
```

Comment:
isn't this unnecessarily complex? why use reflection? wouldn't a simple List of Strings suffice? or a text file listing all resources?

Reply:
I can't see how to implement `getLatestStructureResourceSupplier` or `getAllStructureResourceSuppliers` that way. Do you have a specific approach in mind? If so, please provide a (pseudo-)code snippet.

Reply:
well some file like `database-migrations` with content
```
v15-structure.json
v16-migration.sql
...
```

Or instead some field in `getLatestStructureResourceSupplier` like
```
private static final String RESOURCES = """
v15-structure.json
v16-migration.sql
...
"""
```

just asking, we can leave it as is, but why more complicated than necessary?

Reply:
ok, I forgot about that, all fine ;)